src/System.IO.FileSystem.Primitives/ref/System.IO.FileSystem.Primitives.cs

```diff
@@ -5,7 +5,10 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(Microsoft.Win32.SafeHandles.SafeFileHandle))]
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.IO.FileAccess))]
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.IO.FileMode))]
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.IO.FileShare))]
 
 namespace System.IO
 {
```

Comment:
We now have an entire contract assembly for a single enum?  Should FileAttributes be moved down as well, and then this contract can basically just be for compat?
cc: @weshaggard